.github/workflows/build-fd-image.yaml

```diff
@@ -384,7 +384,7 @@ jobs:
       - build-arm64-base-image-metadata
       - build-arm64-base
     with:
-      source_image: "${{ needs.build-arm64-base-image-metadata.outputs.IMG_NAME }}"
+      source_image: "${{ needs.build-arm64-base-image-metadata.outputs.IMG_NAME }}:${{ needs.build-arm64-base-image-metadata.outputs.version }}"
```

Comment:
This fixes a bug where we were previously promoting `fluentd-latest` on GHCR to "debug" tags on Dockerhub:

```
Promoting ghcr.io/fluent/fluent-operator/fluentd to docker.io/kubesphere/fluentd:latest-arm64-base
```

This fix promotes the proper images:

```
Promoting ghcr.io/***/fluent-operator/fluentd:9.9.9-10-arm64-base to docker.io/kubesphere/fluentd:v9.9.9-10-arm64-base
```